lib/modules/platform/github/index.ts

```diff
@@ -172,7 +177,24 @@ export async function initPlatform({
     renovateUsername,
     token,
   };
-
+  platformResult.hostRules = [];
```

Comment:
can you move the host rule above line 174 ` const platformResult: PlatformResult = {`
and then return the result at the end fully? instead of calculating the platformResult and then changing its host rules?